persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/QueryGenerator.java

```diff
@@ -215,10 +215,10 @@ static PreparedQuery generateVersionQuery() {
   }
 
   /**
-   * Generate a SELECT query to find any entities that have a given realm & parent and that may with
-   * a given location. The check is performed without consideration for the scheme, so a path on one
-   * storage type may give a false positive for overlapping with another storage type. This should
-   * be combined with a check using `StorageLocation`.
+   * Generate a SELECT query to find any entities that have a given realm &amp; parent and that may
+   * with a given location. The check is performed without consideration for the scheme, so a path
```

Comment:
This looks like it got clobbered somehow, it should say `and that may overlap with a given location`

Reply:
updated :+1: